Bottom: 07d0913f84c7487815c000743021a3ab39ad2770
Top:    7ea8a9f770a908799a2b2fb3a2fc6971bc9547c4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-06-05 12:07:58 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 72b6903..74f26d7 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ DrawStrippedText (hidGC gc, ElementType *Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && hid_draw_is_gui (hid_draw)) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& hid_draw_is_gui (hid_draw)*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
